docs/maui/views/DrawingView.md

```diff
@@ -0,0 +1,133 @@
+---
+title: DrawingView - .NET MAUI Community Toolkit
+author: VladislavAntonyuk
+description: "The DrawingView allows to draw lines, save the image and restore it by settings the list of lines."
+ms.date: 04/14/2022
+---
+
+# DrawingView
+
+[!INCLUDE [docs under construction](../includes/preview-note.md)]
+
+The `DrawingView` allows to draw lines, save the image and restore it by settings the list of lines.
+
+## Syntax
+
+The `DrawingView` is invoked using C#.
```

Comment:
I don't quite follow this statement as the immediate example to follow is XAML. What is this referring to?